site/content/docs/5.1/utilities/mixin.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: docs
+title: Utility mixin
+description: Don't want to use utility classes directly in your HTML? Use the `utl()` mixin to compose custom component styles in your source Sass files.
+group: utilities
+aliases: "/docs/5.1/utilities/mixin/"
+toc: true
+---
+
+## How it works
+
+Bootstrap generates hundreds of utilities to quickly and easily style elements through the addition of classes in your HTML. Now with Bootstrap v5.3.0, you can add utilities in your custom Sass with the `utl()` mixin. Using the [utility API](), we generate placeholders for every utility that can be included via Sass mixin. Use the mixin in your own Sass files to include quick styles, override defaults, or compose entire components.
+
+```html
+<!-- Utility classes -->
+<div class="d-inline-flex p-4 mb-md-3">...</div>
+
+<!-- Utility mixin -->
+<div class="test">...</div>
+```
+
+## Motivation
+
+There are two major motivations for creating the utility mixin:
+
+1. **Bootstrap should be as approachable and useful to as many people as possible.** This is why we provide compiled ready-to-go distribution files alongside source Sass and JavaScript. Similarly, not everyone likes pre-built components, and not everyone likes completely utility-driven development.
```

Comment:
Just saying, the choice is not always based on personal preference.
Often you have constraints in the project or ecosystem you work with. E.g. your web framework or CMS might make it difficult to change some of the html, or the html is coming from a 3rd party. So you need to do everything in CSS instead.